libs/filamentapp/src/IBL.cpp

```diff
@@ -136,6 +141,15 @@ bool IBL::loadFromKtx(const std::string& prefix) {
     mSkyboxTexture = Ktx1Reader::createTexture(&mEngine, skyKtx, false);
     mTexture = Ktx1Reader::createTexture(&mEngine, iblKtx, false);
 
+    // TODO: create the fog texture, it's a bit complicated because IBLPrefilter requires
+    //       the source image to have miplevels, and it's not guaranteed here and also
+    //       not guaranteed we can generate them (e.g. texture could be compressed)
+    //IBLPrefilterContext context(mEngine);
```

Comment:
We could check the texture format? Also for gltf_viewer, I don't think we have a way to load a compressed texture really.

Reply:
I would uncomment that code :)

Reply:
okay, I'll get back to it